src/callbacks.rs

```diff
@@ -28,6 +28,16 @@ pub trait RedundantStorageCallback: Send + Sync + Debug {
     /// List objects in the given bucket.
     ///
     /// # Return
-    /// Return a list of object keys present in the bucket.
+    /// Returns a list of object keys present in the bucket.
     fn list_objects(&self, bucket: String) -> Vec<String>;
 }
+
+pub trait LspCallback: Send + Sync {
+    fn channel_information(&self) -> Vec<u8>;
+
+    /// Register a new incoming payment.
+    ///
+    /// # Return
+    /// Returns non empty string with description in case of an error.
+    fn register_payment(&self, bytes: Vec<u8>) -> String;
```

Comment:
We can now handle exceptions with the new version of UniFFI 😃 see [this](https://mozilla.github.io/uniffi-rs/udl/callback_interfaces.html). We should fix this in other callbacks as well.